src/components/general/OutcomeCard.tsx

```diff
@@ -186,8 +170,27 @@ const OutcomeCard = (props: OutcomeCardProps) => {
   // If there is no outcome  value for active year, do not display card set
   if (goalOutcomeValue === undefined) return null;
 
+  const handleClickTab = () => handleClick(node.id);
+
+  const handleKeyDownOnTab = (e: React.KeyboardEvent) => {
+    if (e.code === 'Enter' || e.code === 'Space') {
+      e.preventDefault();
+      handleClick(node.id);
+    }
+  };
+
   return (
-    <CardContainer key={node.id}>
+    <StyledTab
+      key={node.id}
+      role="tab"
+      tabIndex={0}
```

Comment:
I think `tabIndex={-1}` preferred for non-active tabs

Reply:
Hmmm doesn't `tabIndex={-1}` prevent an element from being reached by keyboard navigation?

Reply:
I think this has something to do with expected behaviour of tabpanel
Entering tabpanel takes you to active tab first. Second tab takes you to it's content. If you want to browse tabs inside tabpanel you use arrow keys (I think you need to be in screen reader/keyboard nav mode for that to work)

Roving tabindex
https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/#kbd_roving_tabindex